galley/pkg/server/settings/args.go

```diff
@@ -34,7 +40,7 @@ const (
 )
 
 // Args contains the startup arguments to instantiate Galley.
-type Args struct {
+type Args struct { // nolint:maligned
```

Comment:
Should it be `Instance` now, rather than `Args`?

Reply:
I thought about it but left it as a separate refactoring. I want to do the refactoring of this separately and align it with the naming convention in cmd/server.go with the names used for configuring Viper.

As part of that, most of the names for container structs will change. We can rename it to Instance, or All or something more appropriate at the time.